### DISCUSSION TITLE
Problem with hexamer

### DISCUSSION THREAD

**Discussion by midhunanila** [BartBruininks/mdvwhole#40] [Q&A]
Dear all,
Im using mdvwhole for removing pbc in hexamer. But in dimer its working fine. But in case of hexamer we can see its not working properly. Could ypu please recommend if i need to add any additional flag or something.
Im using the following command  "mdvwhole -f your_gro.gro -x your_xtc.xtc -o whole.xtc" 

Also im attaching the snapshot of the frame it causes the problem with pbc

![vmd](https://github.com/BartBruininks/mdvwhole/assets/78221153/80196e63-88ea-4e13-a683-0547a315a17c)

With Regards,
Midhun
PhD student,
Institute of physics, PAN
Warsaw, poland

**Comment by BartBruininks** [comment]
The problem described here was due to the self contact of the aggregate over PBC, meaning it became an intrinsic periodic object which cannot be completed using MDVWhole. An example of this phenomena is given in the README, where a video shows how the completion of the bilayer ribbon feels under self contact (at the end of the video).

This has been handled in the issues, and over mail. Therefore I regard this as closed. Thanks again Midhun and if things were not resolved please feel free to make another issue.
